src/modules/EditProfile/index.tsx

```diff
@@ -29,10 +29,12 @@ export const EditProfile: FC = () => {
   const loginToken = useSelector(selectToken);
   const userData = useSelector(selectUserData);
 
-  const oldCourses: IOldCourses[] = userData.courses.map((course: Course) => ({
-    ...course,
-    isNew: true,
-  }));
+  const oldCourses: IOldCourses[] = (userData.courses as Course[]).map(
```

Comment:
```
  const oldCourses = userData.courses.map((course: Course) => ({
      ...course,
      isNew: true,
    });
```
And in types.ts in User, in courses field we should leave only Courses[] and remove | [];